onsstove/onsstove.py

```diff
@@ -579,4 +579,9 @@ def lives_saved(self):
 
         total_pop = self.gdf['Calibrated_pop'].sum()
 
-        self.gdf["deaths_avoided"] = self.gdf.apply(lambda row: self.techs[row['final_tech']].deaths_avoided * row['Calibrated_pop'] / total_pop)
\ No newline at end of file
+        self.gdf["deaths_avoided"] = self.gdf.apply(lambda row: self.techs[row['final_tech']].deaths_avoided * row['Calibrated_pop'] / total_pop)
+
+    def extract_time_saved(self):
+
+        self.gdf["time_saved"] = self.gdf.apply(lambda row: self.techs[row['final_tech']].total_time_saved * gdf["Households"])
```

Comment:
In this case we can take the `gdf['Households']` outside the apply:
```python
self.gdf.apply(lambda row: self.techs[row['final_tech']].total_time_saved) * gdf["Households"]
```
That way we make sure that the series of time saved gets multiplied by the series of Households.